### ISSUE TITLE
Accordion: Generic JavaScript

### ISSUE THREAD

**Issue by jbehling-dts** [utahdts/utah-design-system#209]
It would be helpful for the accordion component page to include an example of how to implement the accordion with vanilla JavaScript. An example would be something like this:
```
<div class="accordion">
    <button 
        class="accordion__header button--primary-color button--solid accordion__header--open"
        type="button" 
        onclick="toggleAccordion(this)"
    >
        <h4><span>An Accordion</span></h4>
        <span class="utds-icon-before-circle-chevron-up icon-button__icon"></span>
    </button>
    <div class="accordion__content accordion__content--open">This is some content.</div>
</div>
<script>
    function toggleAccordion(button) {
        const accordionContent = button.nextElementSibling;
        accordionContent.classList.toggle('accordion__content--open');
        button.classList.toggle('accordion__header--open');
        
        const icon = button.querySelector('.icon-button__icon');
        if (button.classList.contains('accordion__header--open')) {
            icon.classList.remove('utds-icon-before-circle-chevron-down');
            icon.classList.add('utds-icon-before-circle-chevron-up');
        } else {
            icon.classList.remove('utds-icon-before-circle-chevron-up');
            icon.classList.add('utds-icon-before-circle-chevron-down');
        }
    }                
</script>
```

**Comment by AustinHaws-Utah** [comment]
Excellent idea! On many of our other component pages we have a "sandbox" that allows you to interact with the component and see the plain html. This works great for most things. But for the Accordion, the javascript is incredibly important and won't appear if we just show the HTML.

We have a JIRA task for implementing the Accordion's sandbox (UDS-360). I've added your script and information to that JIRA task. It looks like there's a good chance it will fit in to our next PI that we are planning in a couple weeks.

**Comment by AustinHaws-Utah** [comment]
Work on this is slated for the start of the next PI at the start of the year!

**Comment by AustinHaws-Utah** [comment]
Currently slated for Sprint 4, finishing Feb 27th

**Comment by AustinHaws-Utah** [comment]
Work for this is complete. Waiting for release.

**Comment by AustinHaws-Utah** [comment]
this is now released in v1.17.0
